test/isEmailAddress.test.js

```diff
@@ -0,0 +1,27 @@
+const { isEmailAddress } = require("../lib");
+
+describe("isEmailAddress", () => {
+  test("Is Email Address (boogie@woogie.com) ?", () => {
+    expect(isEmailAddress("boogie@woogie.com")).toBe(true);
+    expect(isEmailAddress("BOOGIE@WOOGIE.COM")).toBe(true);
+  });
+
+  test("Is Email Address (boogie+label@woogie.com) ?", () => {
+    expect(isEmailAddress("boogie+label@woogie.com")).toBe(true);
+  });
+
+  test("Is Invalid Email Address (boogiewoogie.com) ?", () => {
+    expect(isEmailAddress("boogiewoogie.com")).toBe(false);
+  });
+
+  test("Is Invalid Email Address (boogie @ woogie. noogie) ?", () => {
+    expect(isEmailAddress("boogie @ woogie. fo' noogie")).toBe(true);
```

Comment:
is this supposed to be `toBe(false)` or is the test description incorrect?

Reply:
SHIT. yea. you're right. fixing now. it'll be up in a moment.